west.yml

```diff
@@ -135,7 +135,7 @@ manifest:
       groups:
         - hal
     - name: hal_ti
-      revision: 29af528900d03af02a17b4b7f4fc93f19a81da2a
+      revision: pull/39/head
```

Comment:
Are we still waiting for a PR in Hal?

Reply:
Well, it's a new (small) one enabling the ADC module in CMakeLists :D 
https://github.com/zephyrproject-rtos/hal_ti/pull/39